examples/testing/keystone.ts

```diff
@@ -1,28 +1,28 @@
 import { config } from '@keystone-6/core';
 import { statelessSessions } from '@keystone-6/core/session';
 import { createAuth } from '@keystone-6/auth';
-import { lists } from './schema';
+import { models } from './schema';
 
 // createAuth configures signin functionality based on the config below. Note this only implements
-// authentication, i.e signing in as an item using identity and secret fields in a list. Session
+// authentication, i.e signing in as an item using identity and secret fields in a model. Session
 // management and access control are controlled independently in the main keystone config.
 const { withAuth } = createAuth({
-  // This is the list that contains items people can sign in as
+  // This is the model that contains items people can sign in as
   listKey: 'Person',
   // The identity field is typically a username or email address
   identityField: 'email',
   // The secret field must be a password type field
   secretField: 'password',
 
   // initFirstItem turns on the "First User" experience, which prompts you to create a new user
-  // when there are no items in the list yet
+  // when there are no items in the model yet
   initFirstItem: {
     // These fields are collected in the "Create First User" form
     fields: ['name', 'email', 'password'],
   },
 });
 
-// Stateless sessions will store the listKey and itemId of the signed-in user in a cookie.
+// Stateless sessions will store the modelKey and itemId of the signed-in user in a cookie.
```

Comment:
Assuming this is called a `listKey` in auth should it be `listkey` here?
```suggestion
// Stateless sessions will store the listKey and itemId of the signed-in user in a cookie.
```